### PR TITLE
[CU-2t7r08p] Integration Tests: Staking Rewards

### PR DESCRIPTION
**Pull request by Dom-Roth**
## Issue
https://app.clickup.com/t/2t7r08p

## Description
~~*Please replace this line with info that is not in the ClickUp ticket, eg: "also bumps dependency foo version"*~~

## Checklist

- [x] Implemented all tests
- [x] Implemented all required test checks
- [x] Cleaned up all code
- [x] Added required documentation
- [ ] Ran the whole test suite without any failures
- [x] Ran npm run lint without any warnings / errors

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2t7r08p Integration Tests: Staking Rewards](https://app.clickup.com/t/2t7r08p)

**Comment by Dom-Roth** [comment]
Waiting for completion of the current pallet rework.

File: code/integration-tests/runtime-tests/test/tests/stakingRewards/stakingRewardsTests.ts [hunk 1]
```diff
@@ -0,0 +1,2354 @@
...
+      await waitForBlocks(api);
+      const result = await sendWithBatchAndWaitForSuccess(
+        api,
+        walletPoolOwner,
+        api.events.stakingRewards.RewardsPotIncreased.is,
+        [
+          api.tx.stakingRewards.addToRewardsPot(poolIds[0], assetIds[0], amount, keepAlive),
+          api.tx.stakingRewards.addToRewardsPot(poolIds[1], assetIds[1], amount, keepAlive),
+          api.tx.stakingRewards.addToRewardsPot(poolIds[2], assetIds[2], amount, keepAlive)
+          //api.tx.stakingRewards.addToRewardsPot(poolIds[3], assetIds[3], amount, keepAlive)
```
Comment (kollegian) [inline code/integration-tests/runtime-tests/test/tests/stakingRewards/stakingRewardsTests.ts:950]: commented out

**Review by kollegian — APPROVED** [review]

File: code/integration-tests/runtime-tests/test/tests/stakingRewards/stakingRewardsTests.ts (hunk 1 above)
Comment (Dom-Roth) [reply code/integration-tests/runtime-tests/test/tests/stakingRewards/stakingRewardsTests.ts]: Fixed! Thank you sir. 👍

**Review by Martin-Kiss — APPROVED** [review]

**Pull request merged**